release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -682,6 +710,22 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* In {product-title} 4.6, the custom metrics API server provides support for custom metrics. The custom metrics API server does not implement the OpenAPI specification and the following messages are sent to the `kube-apiserver` logs every 60 seconds:
```

Comment:
replace `custom metrics API server` with `resource metrics API server`

Reply:
Thank you Sergiusz for your review! I have now implemented this suggestion.